integration/containerd/cri/integration-tests.sh

```diff
@@ -18,6 +18,7 @@ RUNTIME=${RUNTIME:-containerd-shim-kata-v2}
 SHIMV2_TEST=${SHIMV2_TEST:-""}
 FACTORY_TEST=${FACTORY_TEST:-""}
 KILL_VMM_TEST=${KILL_VMM_TEST:-""}
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
```

Comment:
Hmm. I wonder whether not setting the KATA_HYPERVISOR was proposital or a bug that wasn't caught during coding / reviewing.
@GabyCT, @jcvenegas, do you know?

Reply:
On commit fa5b6af it was introduced the conditional to skip the test in case of KATA_HYPERVISOR is clh. The CI jobs which test clh will set KATA_HYPERVISOR so it works. But for default jobs, i.e. to test with qemu, that variable might not be set on env (some scripts assume the default is "qemu"). Thus, I believe it is a bug. Anyway, Gaby and Carlos will know for sure.